src/pages/home/Admin/RequestDetail.js

```diff
@@ -0,0 +1,143 @@
+import * as React from "react";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import {Chip} from "@mui/material";
+import Table from "@mui/material/Table";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableRow from "@mui/material/TableRow";
+import Paper from "@mui/material/Paper";
+import "./RequestDetail.css";
+import {useLocation} from "react-router-dom";
+
+function RequestDetail() {
+  const location = useLocation();
+  const rowData = location.state?.rowData;
+
+  const getStatusColor = (status) => {
+    if (status === "APPROVED") {
+      return "primary";
+    } else if (status === "PENDING") {
+      return "error";
```

Comment:
Could you change this "error" to "warning" for PENDING status?

Reply:
That is the default label value  for chip element, which cant be changed